/*
 * ProjectTemplate
 *
 * Created by artembirmin on 2/5/2022.
 */

package com.incetro.secondapplication.model.data.network


class NetworkConnectionError : Throwable()